.github/workflows/release.yaml

```diff
@@ -351,6 +353,9 @@ jobs:
           tag: ${{ env.VERSION }}
           overwrite: true
 
+{%- endraw -%}
+{%- if itch_username != "" -%}
```

Comment:
I'd remove this `if`. The workflow already handles a commented out `ITCH_TARGET` correctly. If we leave it in, a user can still add itch support later.